data/remotes.d/README

```diff
@@ -0,0 +1,5 @@
+Steps to make vendor firmware work:
+
+* Change /etc/fwupd/remotes.d/vendor.conf to have `Enabled=true`
+* Add the metadata to /usr/share/fwupd/remotes.d/vendor/vendor.xml.gz
```

Comment:
Since a sample vendor.xml.gz isn't included, some extra comments about the expectations for appstream format (or even linking to online documentation about how to make it) will make this clearer to someone creating metadata for the first time.